foreman/data_refinery_foreman/surveyor/test_harmony.py

```diff
@@ -271,48 +280,67 @@ def test_geo_harmony(self):
         """
         Thoroughly tests a specific GEO harmonization
         """
-
         # Weird ones caused bugs
         gse = GEOparse.get_GEO("GSE94532", destdir="/tmp/GSE94532/", silent=True)
-        preprocessed_samples = preprocess_geo(gse.gsms.items())
+        for _, sample in gse.gsms.items():
```

Comment:
What are we checking for here?

Reply:
From the comment I assume that calling preproces_geo() used to cause errors on this experiment until it got fixed?

It'd be nice if the comment was a little bit more descriptive... I'll add a couple checks of some sort here.